launcher/image/testworkloads/customtoken/happypath/main.go

```diff
@@ -0,0 +1,230 @@
+// package main is a binary that will print out the validation status of a custom attestation token.
+package main
+
+import (
+	"context"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+const (
+	socketPath     = "/run/container_launcher/teeserver.sock"
+	expectedIssuer = "https://confidentialcomputing.googleapis.com"
+	wellKnownPath  = "/.well-known/openid-configuration"
+)
+
+type jwksFile struct {
+	Keys []jwk `json:"keys"`
+}
+
+type jwk struct {
+	N   string `json:"n"`   // "nMMTBwJ7H6Id8zUCZd-L7uoNyz9b7lvoyse9izD9l2rtOhWLWbiG-7pKeYJyHeEpilHP4KdQMfUo8JCwhd-OMW0be_XtEu3jXEFjuq2YnPSPFk326eTfENtUc6qJohyMnfKkcOcY_kTE11jM81-fsqtBKjO_KiSkcmAO4wJJb8pHOjue3JCP09ZANL1uN4TuxbM2ibcyf25ODt3WQn54SRQTV0wn098Y5VDU-dzyeKYBNfL14iP0LiXBRfHd4YtEaGV9SBUuVhXdhx1eF0efztCNNz0GSLS2AEPLQduVuFoUImP4s51YdO9TPeeQ3hI8aGpOdC0syxmZ7LsL0rHE1Q",
```

Comment:
Yes you are right, N is the modulus and E is the exponent. However, I don't think having those as comments is useful to understanding what this code does. Given that this code is converting from one value to the other I felt it would be more useful to have an example of what N and E look like.